encoding/video-artifacts.md

```diff
@@ -261,6 +261,21 @@ it's better to do this under 16bit to avoid precision problems.
 
 ![](images/overflow_notice.jpg)
 
+In this case, the process under 8bit may lead to
+banding and other weird artifact.
+We should avoid doing these scaling under low precision.
+
+Here's some examples if you want to use `std.Expr()`.
+
+```py
+# We suppose "src16" is a 16bit video clip and we only process Y plane in this example as well as bypass the UV planes.
+# I have to underline that this example is processed under 16bit and I don't recommend to do this at any lower precisions.
+# If you want to process UV plane, you need to take care of these linear map expressions because the upper and lower limits in UV planes are different.
+YC = core.std.Expr(src16, expr=["x 255 / 219 * 4096 +",""], format=vs.YUV)   #this makes [0-255] to [16-235] at y plane
+YC = core.std.Expr(src16, expr=["x 4096 - 219 239 / * 4096 +",""], format=vs.YUV)   #this makes [16-255] to [16-235] at y plane
+YC = core.std.Expr(src16, expr=["x 219 255 / * 4096 +",""], format=vs.YUV)   #this makes [0-235] to [16-235] at y plane
```

Comment:
This is functionally equivalent to the first one. Shouldn’t it rather be
```py
YC = core.std.Expr(src16, expr=["x 219 235 / * 4096 +",""], format=vs.YUV)
```
?

Reply:
Nope, it should be "x / 235 * 219 + 4096" or "x 235 / 219 * 4096 +" in reverse polish notation.

I think I made a mistake with the expression.

Reply:
Thanks! Fixed

Reply:
Actually, wouldn’t it make more sense to use the built-in [``std.Levels``](http://www.vapoursynth.com/doc/functions/levels.html) filter instead of the whole Expr business?

Reply:
Wow, that looks really useful. I wonder why it doesn't accept lists for the `*_in` and `*_out` parameters that are expanded to the number of planes to be processed, like many other functions. Although the ones I'm thinking of probably accept multiple clips.